src/components/exercises/SyllableTable.stories.js

```diff
@@ -6,7 +6,8 @@ import SyllableTable from './SyllableTable';
 storiesOf('exercises/SyllableTable', module).add('ne', () => (
   <SyllableTable
     sound={require('../../assets/sounds/ne.mp3')}
-    syllables={['na', 'ne', 'no', 'sa', 'se', 'so', 'ta', 'te', 'to']}
-    index={1}
+    letters={['n', 's', 't']}
```

Comment:
Can you please also create a story and snapshot test for the case of https://abc.serlo.org/try/#524 (and maybe interesting cases before that?). So that one can review the UI and usage for that?